samples/bluetooth/central_uart/sysbuild.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC=y
```

Comment:
This will cause error for single core target. We need to find another way. Maybe it's possible to leave only sysbuild_bt_rpc.conf?

Reply:
Yes, leaving only sysbuild_bt_rpc.conf